examples/pod_api.rs

```diff
@@ -11,15 +10,15 @@ use kube::{
 #[tokio::main]
 async fn main() -> anyhow::Result<()> {
     std::env::set_var("RUST_LOG", "info,kube=debug");
-    env_logger::init();
+    tracing_subscriber::fmt::init();
```

Comment:
This example can be used to see the output from `TraceLayer` for the default client.

![image](https://user-images.githubusercontent.com/639336/120424802-eba92500-c321-11eb-90ac-02608b5dcc72.png)

Reply:
I think that looks great, and probably sufficient to rip out all the `#[instrumen]` instructions in the `Api`.

One philosophical point; having the duration as a separate log line does feel to me like it's encroaching on what should be tracing's turf:
![Screenshot from 2021-06-02 13-45-20](https://user-images.githubusercontent.com/134092/120482555-2609d500-c3a9-11eb-9869-6c9f0732fa89.png)
A span viewer will always have the timing information anyway, whereas a line for it here is just a harder way to read this, and it imo pollutes the log space.